guides/testing-with-test-dispatch.md

```diff
@@ -0,0 +1,76 @@
+# Testing with TestDispatch
+
+This library was build to test if forms and links have the expected results in
+controller tests.
+
+It can be used with or without
+[TestSelector](https://github.com/DefactoSoftware/TestSelector)
```

Comment:
But it is required as dependency right?

Reply:
Yeah, but it doesn’t have to be implemented in the views to use this.

Reply:
We could also make the dependency optional I guess

Reply:
Changed the text a bit

Reply:
> We could also make the dependency optional I guess

Yeh was thinking of that as well